src/functions/template-tags/general.php

```diff
@@ -1135,7 +1135,7 @@ function tribe_get_days_between( $start_date, $end_date, $day_cutoff = '00:00' )
 			$end_date->modify( '-1 day' );
 		}
 
-		return Tribe__Events__Date_Utils::dateDiff( $start_date->format( 'Y-m-d ' . $day_cutoff ), $end_date->format( 'Y-m-d ' . $day_cutoff ) );
+		return Tribe__Events__Date_Utils::date_diff( $start_date->format( 'Y-m-d ' . $day_cutoff ), $end_date->format( 'Y-m-d ' . $day_cutoff ) );
```

Comment:
I added the additional files because I found `dateDiff` to `date_diff` changes in the previous PR.